Documentation/install/upgrade.rst

```diff
@@ -844,27 +812,16 @@ Affected versions
 When a CIDR policy with too many unique prefix lengths is imported, Cilium will
 reject the policy with a message like the following:
 
-.. tabs::
-  .. group-tab:: Cilium 1.0
-
-    .. code-block:: shell-session
-
-     $ cilium policy import too_many_cidrs.json
-     Error: Cannot import policy: [PUT /policy][500] putPolicyFailure  too many
-     egress CIDR prefix lengths (128/40)
-
-  .. group-tab:: Cilium 1.1 or later
-
-     .. code-block:: shell-session
+.. code-block:: shell-session
 
-        $ cilium policy import too_many_cidrs.json
-        Error: Cannot import policy: [PUT /policy][500] putPolicyFailure  Adding
-        specified prefixes would result in too many prefix lengths (current: 3,
-        result: 32, max: 18)
+  $ cilium policy import too_many_cidrs.json
```

Comment:
nit: Looks like this is only two spaces. Sphinx rst files typically use three-space indentation. :bowling: